reUnit/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "reunit",
```

Comment:
Yeah, but I want to work with @glennsl to integrate it better with jest first before encouraging adoption. The idea would be to provide a common DSL that can be used with either a native test runner, or jest for a more full featured experience.